pkgs/tools/networking/n2n/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "n2n";
+  version = "2.8";
+
+  src = fetchFromGitHub {
+    owner = "ntop";
+    repo = "n2n";
+    rev = version;
+    hash = "sha256-2xJ8gYVZJZoKs6PZ/9GacgxQ+/3tmnRntT1AbPe1At4=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  postPatch = ''
+    patchShebangs autogen.sh
+  '';
+
```

Comment:
```suggestion

  preAutoreconf = ''
    ./autogen.sh
  '';

```

Reply:
This runs the configure script twice, since `autogen.sh` already calls `autoreconf`. I've applied the suggestion but consider this suboptimal.